.github/workflows/ci.yml

```diff
@@ -8,15 +8,17 @@ on:
   push:
     branches:
       - main
-    paths:
-      - '**'
-      - '!docs/**'
+    # TODO: TESTING CI
```

Comment:
@carlos-kelly @gksander -- So the path `**` **should** be firing for our changes here causing CI to run. Commenting this out is causing CI to run, but I'm not sure _why_ that's happening.

Reply:
Just overrode your commit using [this guide](https://github.blog/changelog/2019-09-30-github-actions-event-filtering-updates/):

```yml
paths-ignore:
      - "docs/**"
```

which seems to be the appropriate equivalent of what we're trying to do. Seems to work (CI is triggered), and feels like a more expressive syntax  🎈